### PR TITLE
Delete and Add functionality

### PR DESCRIPTION
**Pull request by tkz96**
I have modified the code to:

- Dynamically add and delete tasks
- Use Local Storage to storage the data

### CODE REVIEW

File: src/template.html [hunk 1]
```diff
@@ -1,29 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
```
Comment (SajjadAhmad14) [inline src/template.html:3]: Please add the `start` script to the `package.json` file and use `webpack` to start the local server. Your app does not start due to this error.
error

File: src/Tasks.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  }
+}
+
+const populateList = (tasksArray) => {
+  todoList.innerHTML = '';
+  for (let i = 0; i < tasksArray.length; i += 1) {
+    const li = document.createElement('li');
+    li.id = `${tasksArray[i].index}`;
+    li.innerHTML = `<input type="checkbox"><div class="task-text" contenteditable="true" data_id='${i}'>${tasksArray[i].description}</div>
+        <i class="fas fa-trash-alt delete" data_id='${i}'></i>`;
```
Comment (SajjadAhmad14) [inline src/Tasks.js:19]: Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
Kindly see the screenshot below which shows that indexes are not unique after deleting an index.
indexes

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Talha, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Dipeshtwis — APPROVED** [review]
Hi @tkz96, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**